packages/story-editor/src/utils/useCORSProxy.js

```diff
@@ -42,7 +42,7 @@ function useCORSProxy() {
    *
    * @type {function(): boolean}
    */
-  async function checkResourceAccess(link) {
+  const checkResourceAccess = useCallback(async (link) => {
```

Comment:
@swissspidy Any concerns with this change here?

Apparently this was a dependency for `generateBaseColor` which is a dep for `insertElement` which caused `onInsert` to be updated every time when something happened which also caused the `InnerElement` etc. re-rendering all the time.

Reply:
No concerns.

I'm actually looking into removing `generateBaseColor` from `insertElement` anyway.

Reply:
>I'm actually looking into removing generateBaseColor from insertElement anyway. 

Are you? Why, it is need for third party resources that don't have base color.

Reply:
@spacedmonkey I'll share more information soon. **tl;dr:** `useInsertElement` is the wrong place to do this.

Reply:
See #9907